### DISCUSSION TITLE
Serving static files

### DISCUSSION THREAD

**Discussion by mybardaklar** [webdiscus/html-bundler-webpack-plugin#89] [Q&A]
How can i serve static files? For example: i would like to create "public" folder and i would like to use it as a static files folder. Is it possible? If it is, i would love to learn.

Thank you! 😺

**Comment by webdiscus** [comment]
> How can i serve static files

What do you mean? 
Do you will use static images from `public` directory in HTML template and these static files should not be processed?
Also shortly: how to disable processing for some static resources?

**Reply by webdiscus** [reply]
Url values are not be processed:
```
src="https://example.com/img/image.png"
src="//example.com/img/image.png"
src="/img/image.png"
```

If your static image files are in the `public/img/`, e.g. `public/img/image.png`, then use the static route to the file with forwading `/` slash:

```html
<img src="/img/image.png" > <!-- static image file will not be processed, stay as is -->
```

```html
<img src="./img/image.png" > <!-- the relative image file will be processed via webpack -->
```

See please the docs for [sources](https://github.com/webdiscus/html-bundler-webpack-plugin/discussions/89#loader-option-sources) (processing of custom tag attributes) option.
